src/Controllers/BaseController.php

```diff
@@ -36,8 +53,8 @@ abstract class BaseController implements Controller
 	{
 		try {
 			return $this->context->addBox($type);
-		} catch (\Throwable $e) {
-			throw new \RuntimeException($e->getMessage(), $e->getCode(), $e);
+		} catch (Throwable $e) {
```

Comment:
Why is this better?

Reply:
I have no exact opinion about that. Autofixers do that because it "simplifies" FQN like other classes in namespaces.

Reply:
We have agreed, so it's resolved.